File: Core/Entities/Order.cs
namespace Core.Entities
{
    public class Order : BaseEntity
    {
        public string idClient { get; set; }
        public CustomerBasket customerBasket { get; set; } 
        public int customerBasketId { get; set; }
        public int shippingPrice { get; set; }
        public string  date { get; set; }
        public decimal total { get; set; }
        public string nameClient { get; set; }
        public string adressClient { get; set; }
        public string  numeroClient { get; set; }
    }
}

File: client/src/app/shared/models/order.ts
import { IBsaket } from "./basket";

export interface IOrder {
    basketId: number;
    shippingPrice: number;
    date: string;
}
export class Order {
    basketId: number;
    shippingPrice: number;
    date: string;
    total: number;
}
export interface IOrderDis {
    id: number;
    idClient: string;
    customerBasket: IBsaket;
    customerBasketId: number;
    shippingPrice: number;
    date: string;
    total: number;
    nameClient: string;
    adressClient: string;
    numeroClient: string;
    state:string;
}